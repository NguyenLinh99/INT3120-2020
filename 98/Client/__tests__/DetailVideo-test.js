import 'react-native';
import React from 'react';

import renderer from 'react-test-renderer';
import DetailVideo from '../screens/VideoHD/DetailVideo';

const vid ='https://truongdaotaolaixehcm.com/wp-content/uploads/2017/03/Thi-B2-Ch%C3%ADp-Sa-H%C3%ACnh-m%E1%BB%9Bi-nh%E1%BA%A5t-2018.mp4?_=1'

it('DetailVideo snapshot', () => {
    const snap = renderer.create(
        <DetailVideo vid={vid} />
    ).toJSON();
    expect(snap).toMatchSnapshot();
});