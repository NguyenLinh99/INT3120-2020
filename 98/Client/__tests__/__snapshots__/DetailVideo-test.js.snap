// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailVideo snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "overflow": "hidden",
      },
      undefined,
    ]
  }
>
  <RNCWebView
    cacheEnabled={true}
    javaScriptEnabled={true}
    messagingEnabled={false}
    onContentProcessDidTerminate={[Function]}
    onHttpError={[Function]}
    onLoadingError={[Function]}
    onLoadingFinish={[Function]}
    onLoadingProgress={[Function]}
    onLoadingStart={[Function]}
    onMessage={[Function]}
    onShouldStartLoadWithRequest={[Function]}
    source={
      Object {
        "uri": "https://truongdaotaolaixehcm.com/wp-content/uploads/2017/03/Thi-B2-Ch%C3%ADp-Sa-H%C3%ACnh-m%E1%BB%9Bi-nh%E1%BA%A5t-2018.mp4?_=1",
      }
    }
    style={
      Array [
        Object {
          "flex": 1,
          "overflow": "hidden",
        },
        Object {
          "backgroundColor": "#ffffff",
        },
        Object {
          "flex": 1,
        },
      ]
    }
    useSharedProcessPool={true}
  />
</View>
`;
